index.d.ts

```diff
@@ -231,6 +232,35 @@ declare module 'warframe-items' {
         corrosive?: number;
         radiation?: number;
     }
+    
+    interface Attack {
+        name: string;
+        duration?: number;
+        radius?: number;
+        speed?: number;
+        pellet?: Pellet;
+        crit_chance?: number;
+        crit_mult?: number;
+        status_chance?: number;
+        charge_time?: number;
+        shot_type?: string;
+        shot_speed?: number;
+        flight?: number | string;
+        falloff?: Falloff;
+        damage: DamageTypes;
+    }
+    
+    interface SlamAttack {
```

Comment:
Is there a real need for SlamAttack to be seperated from Attack? As there's other attack types which does not have a separated interface for it.

Reply:
The others don't have radically different formats for the source data

Reply:
I would honestly do a melee one, but I don't know how it works for doing things that extend others.